srcpkgs/compton/template

```diff
@@ -21,5 +22,6 @@ pre_build() {
 
 post_install() {
 	vlicense LICENSES/MIT
+	vlicense LICENSES/MPL-2.0
```

Comment:
it isn't required to provide the MPL license file

Reply:
Fixed.